invenio.cfg

```diff
@@ -336,8 +353,16 @@ RDM_CUSTOM_FIELDS = [
     KeywordCF(name='openbiodiv:TaxonomicConceptLabel', multiple=True),
     # obo
     RelationshipListCF(name='obo:RO_0002453'),
+    # meeting
+    *MEETING_CUSTOM_FIELDS,
 ]
 
+# Custom fields UI components
+RDM_CUSTOM_FIELDS_UI = [
+    MEETING_CUSTOM_FIELDS_UI
```

Comment:
Will that conflict with the journal config PR?

Reply:
Yes, I will rebase it though when things are merged.